paasta_tools/list_kubernetes_service_instances.py

```diff
@@ -88,9 +102,25 @@ def main():
         else:
             app_name = compose_job_id(name, instance)
         service_instances.append(app_name)
-    print("\n".join(service_instances))
+    if args.shuffle:
+        random.shuffle(service_instances)
+    if args.group_lines:
+        group_lines(service_instances, args.group_lines)
+    else:
+        print("\n".join(service_instances))
```

Comment:
This doesn't seem to ever get around to printing the instances if shuffle or group_lines is specified since the print is in an `else` here

Reply:
jk i have now read group_lines and see we're printing down there.